drive/files.py

```diff
@@ -95,18 +97,20 @@ def unlink(self):
         # https://developers.google.com/drive/v3/reference/files/delete
         return self._client.remove_file(self.id) == ""
 
-    def rename(self, new_name):
+    def rename(self, new_name: str) -> None | Literal[False]:
```

Comment:
This return type is kinda weird! It is how things behave, though: `False` if no client and `None` otherwise (regardless of whether anything was updated).

Reply:
Agreed; I’ll first cut a release with the current behavior and then make another one with breaking changes to fix these things.